core/src/main/java/org/itsallcode/openfasttrace/core/importer/MultiFileImporterImpl.java

```diff
@@ -131,12 +132,18 @@ public MultiFileImporter importRecursiveDir(final Path dir, final String glob)
         return this.specItemBuilder.build();
     }
 
-    private Importer createImporter(final InputFile file, final SpecificationListBuilder builder)
+    private Optional<Importer> createImporterIfPossible(final InputFile file, final SpecificationListBuilder builder)
     {
-        final ImporterFactory importerFactory = this.factoryLoader.getImporterFactory(file);
-        final Importer importer = importerFactory.createImporter(file, builder);
-        LOG.fine(() -> "Created importer of type '" + importer.getClass().getSimpleName()
-                + "' for file '" + file + "'");
+        final Optional<ImporterFactory> importerFactory = this.factoryLoader.getImporterFactory(file);
+        final Optional<Importer> importer = importerFactory.isPresent() ?
+                Optional.of(importerFactory.get().createImporter(file, builder)) :
+                Optional.empty();
+
+        LOG.fine(() -> ( importer.isPresent() ?
+                            "Created importer of type '" + importer.getClass().getSimpleName() :
+                            "No import" )
```

Comment:
```suggestion
        LOG.fine(() -> (importer.isPresent() ?
                            "Created importer of type '" + importer.getClass().getSimpleName() :
                            "No import")
```
Please use auto-formatting. that should also make sure that all spaces are uniform.

Reply:
DONE